packages/camera/tizen/src/camera_device.h

```diff
@@ -323,9 +321,16 @@ class CameraDevice {
   bool UnsetRecorderRecordingLimitReachedCb();
   void UpdateStates();
 
+  bool RegistAvailableMediaPacket(media_packet_h packet);
+  media_packet_h UnregistAvailableMediaPacket();
```

Comment:
Regist → Register?

Reply:
it is no longer a member, please review again